static/src/js/components/ContactInfo/ContactInfo.scss

```diff
@@ -0,0 +1,25 @@
+fieldset.account-form {
```

Comment:
I would like to see this CSS and the component `classNames` BEMified like our other components. I can help with this.